src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201508081500.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle;
+
+use PHPCR\Migrations\VersionInterface;
+use PHPCR\NodeInterface;
+use PHPCR\Query\QueryManagerInterface;
+use PHPCR\SessionInterface;
+use Sulu\Component\Content\Compat\Structure;
+use Sulu\Component\Content\Document\RedirectType;
+use Sulu\Component\Content\Metadata\Factory\StructureMetadataFactoryInterface;
+use Sulu\Component\Content\Metadata\PropertyMetadata;
+use Sulu\Component\Content\Metadata\StructureMetadata;
+use Sulu\Component\DocumentManager\PropertyEncoder;
+use Sulu\Component\PHPCR\SessionManager\SessionManagerInterface;
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class Version201508081500 implements VersionInterface, ContainerAwareInterface
```

Comment:
Can you add a PHPDoc? Hard to get a quick idea of what this migration is doing without it.